CHANGELOG.md

```diff
@@ -10,7 +10,20 @@ The semantic versioning started from version 0.2.1.
 
 _No documentation available about unreleased changes as of yet._
 
-## [0.4.0](https://github.com/infinum/coding-standards-wp/compare/0.3.1...0.4.0) - 2018-09-30
+## [0.4.1](https://github.com/infinum/coding-standards-wp/compare/0.3.1...0.4.1) - 2018-11-15
```

Comment:
+1 for writing detailed documentation, I'll assume it's because of me and take all the credit. 😄

Reply:
You know it 😆